build.sbt

```diff
@@ -133,40 +140,44 @@ lazy val `support-config` = (project in file("support-config"))
 lazy val `support-services` = (project in file("support-services"))
   .configs(IntegrationTest)
   .settings(
-    commonSettings,
+    releaseSettings,
     integrationTestSettings,
     libraryDependencies ++= commonDependencies
-  ).dependsOn(`support-internationalisation`, `support-models`, `support-config`, `support-rest`)
-  .aggregate(`support-internationalisation`, `support-models`, `support-config`, `support-rest`)
+  ).dependsOn(`support-internationalisation`, `support-models`, `support-config`, `module-rest`, `module-aws`)
+  .aggregate(`support-internationalisation`, `support-models`, `support-config`, `module-rest`, `module-aws`)
```

Comment:
I went for "module" rather than "support" as the prefix, to separate them from the standard layered approach we used for many of the modules.  Happy for other conventions, as long as we avoid a wall of `support-*` with no obvious high level structure.

Reply:
module works for me